pkg/cloud/services/ec2/instances.go

```diff
@@ -512,6 +512,10 @@ func (s *Service) runInstance(role string, i *infrav1.Instance) (*infrav1.Instan
 			ebsRootDevice.Iops = aws.Int64(i.RootVolume.IOPS)
 		}
 
+		if i.RootVolume.Throughput != 0 {
```

Comment:
We should probably add a check in the validating webhook to stop people entering negative numbers. What do you think?

Reply:
Added, an additional test to ensure gp2 volumes do not set throughput.